src/main/scala/com/gu/acquisition/services/GAService.scala

```diff
@@ -22,14 +27,36 @@ private[services] class GAService(implicit client: OkHttpClient)
   private val gaPropertyId: String = "UA-51507017-5"
   private val endpoint: HttpUrl = HttpUrl.parse("https://www.google-analytics.com")
 
-  private[services] def buildBody(submission: AcquisitionSubmission): Either[BuildError, RequestBody] = {
-    buildPayload(submission).map{
-      payload =>
-        RequestBody.create(null, payload)
+  private[services] def buildBody(submission: AcquisitionSubmission)(implicit ec: ExecutionContext): EitherT[Future, BuildError, RequestBody] = EitherT {
+    getAnnualisedValue(submission.acquisition)
+      .fold(
+        error => {
+          logger.warn(s"Couldn't retrieve annualised value for this acquisition: $error")
+          buildPayload(submission, 0) // We still want to record the acquisition even if we can't get the AV
+        },
+        value => buildPayload(submission, value)
+      ).map {
+      maybePayload =>
+        maybePayload.map { payload =>
+          logger.debug(s"GA payload: $payload")
+          RequestBody.create(null, payload)
+        }
     }
   }
 
-  private[services] def buildPayload(submission: AcquisitionSubmission, transactionId: Option[String] = None): Either[BuildError, String] = {
+  private def getAnnualisedValue(acquisition: Acquisition)(implicit ec: ExecutionContext) = {
+    val acquisitionModel = AcquisitionModel(acquisition.amount,
```

Comment:
Can an `apply` method be added to `AcquisitionModel` that takes and instance of `Acquisition `  to handle the instantiation?

Reply:
`AcquisitionModel` is defined in a library which doesn't know about `Acquisition`s so not in this case